Nginx_Auth_Proxy/confd/templates/nginx.conf.tpl

```diff
@@ -94,6 +87,8 @@ http {
         # Use API endpoint in user-service for checking authentication
         location /auth {
             limit_except      GET POST PUT { deny all; }
+            # include /etc/includes/without-auth-headers.conf;
```

Comment:
was this meant to be commented out?

Reply:
Thank you Mihai, I commented it out while I was checking something. Forgot to uncomment.
It's uncommented now.